### PR TITLE
Internationalization and Russian translation

### PR DESCRIPTION
**Pull request by joe513**
#3 Fix

### CODE REVIEW

**Comment by joe513** [comment]
We also need to add a language switcher.

File: playlist/settings.py [hunk 1]
```diff
@@ -146,3 +147,10 @@
         from playlist.production import *
     except ImportError:
         pass
+
+from django.utils.translation import gettext_lazy as _
```
Comment (inoks) [inline playlist/settings.py]: Please move this import to the top of the file.

File: templates/_base.html [hunk 2]
```diff
@@ -1,7 +1,8 @@
 <!doctype html>
 <html lang="en">
 <head>
-  <title>{% block title %}M3U8 TV Channels Playlist Editor{% endblock %}</title>
+    {% load i18n %}
```
Comment (inoks) [inline templates/_base.html]: Please always add {% load ... %} in the beginning of the template (after extends tag if they exists).

File: playlist/urls.py [hunk 3]
```diff
@@ -3,14 +3,18 @@
 from django.contrib.auth import views as auth_views
 from django.conf import settings
 from django.conf.urls.static import static
+from django.conf.urls.i18n import i18n_patterns
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+               ]
+
+urlpatterns += i18n_patterns(
```
Comment (inoks) [inline playlist/urls.py]: Can you instead provide language switch form in the header of the pages (inside base template)?

Comment (joe513) [reply playlist/urls.py]: I'll try

File: playlist/settings.py (hunk 1 above)
Comment (joe513) [reply playlist/settings.py]: "In the same" means in this repository or my forked repository?

**Pull request closed without merging**